blackjax/inference/trajectory.py

```diff
@@ -50,21 +50,15 @@ class Trajectory(NamedTuple):
     num_states: int
 
 
-def append_to_trajectory(
-    direction: int, trajectory: Trajectory, state: IntegratorState
+def rightmost_append_to_trajectory(
```

Comment:
For every language I know of  `append`  = `add to the right` so we can probably name it `append_to_trajectory`.